x/oracle/types/validatorWhitelist.go

```diff
@@ -0,0 +1,92 @@
+package types
+
+import (
+	sdk "github.com/cosmos/cosmos-sdk/types"
+)
+
+// ValidatorWhitelist define the address and its power
+type ValidatorWhitelist struct {
+	Whitelist map[string]uint32 `json:"whitelist"`
```

Comment:
I think we should be mapping a string to a string to a uint32 instead of just a string to a uint32.

Reply:
The data structure is for one network. please check the oracle/keeper/validatorWhiteList.go, it maintains an object of the type for each networkDescriptor.